build-system/tasks/runtime-test.js

```diff
@@ -219,21 +214,18 @@ gulp.task('test', 'Runs tests',
       testFiles = testFiles.concat(glob.sync(testPaths[index]));
     }
 
-    if (argv.randomize) {
-      testFiles = shuffleArray(testFiles);
+    if (argv.randomize || argv.a4a) {
+      const seed = argv.seed || Math.random();
+      util.log(
+          util.colors.red('Randomizing:'),
+          yellow('Seeding with value', seed));
+      util.log(util.colors.red('Randomizing:'),
```

Comment:
nit: Red is typically used to indicate an error. I'd suggest using yellow for the text and cyan for the actual arguments, in keeping with the patterns used elsewhere in this file. Here's a suggested rewording:

```
util.log(
    util.colors.yellow('Randomizing: Seeding with value'),
    util.colors.cyan(seed));
util.log(
    util.colors.yellow('To rerun tests in the same order, append'),
    util.colors.cyan('--seed=${seed}'),
    util.colors.yellow('to your invocation of'),
    util.colors.cyan('gulp test'));
```

Reply:
Or green if you don't like yellow :)

Reply:
done